src/Checkbox/checkbox.style.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import { error } from "../Colors";
+
+const CheckboxStyle = styled.div`
+  display: flex;
+  align-items: center;
+
+  label {
+    padding: 0;
+    font-size: 24px;
+    font-weight: normal;
+    line-height: 1.38;
+    letter-spacing: normal;
+    color: #c9c9c9;
+    margin-right: 16px;
+    cursor: pointer;
+  }
+
+  input[type="checkbox"] {
+    display: none;
+  }
+
+  input[type="checkbox"] + label:after {
+    font-family: "vivy-icons";
+    content: "\\e80f";
```

Comment:
can import it as variable from the `public/fonts/vivy-icons/iconMapping.json` file we have?
This way if we update the vivy-icons at some point this will be updated as well and wont break if `e80f ` is not valid anymore.